tests/hw-accel/amdgpu/basic/tests/basic-test.go

```diff
@@ -0,0 +1,242 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-gotests/tests/hw-accel/amdgpu/internal/deviceconfig"
+	"github.com/openshift-kni/eco-gotests/tests/hw-accel/amdgpu/internal/get"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/amdgpu"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	tsp "github.com/openshift-kni/eco-gotests/tests/hw-accel/amdgpu/basic/internal/tsparams"
+	"github.com/openshift-kni/eco-gotests/tests/internal/inittools"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("AMD GPU Basic Tests", Ordered, Label(tsp.LabelSuite), func() {
+
+	Context("AMD GPU Basic 01", Label(tsp.LabelSuite+"-01"), func() {
+
+		apiClient := inittools.APIClient
+
+		amdListOptions := metav1.ListOptions{
+			LabelSelector: fmt.Sprintf("%s=%s", tsp.AMDNFDLabelKey, tsp.AMDNFDLabelValue),
+		}
+
+		amdNodeBuilders, amdNodeBuildersErr := nodes.List(apiClient, amdListOptions)
+
+		BeforeAll(func() {
+
+			Expect(amdNodeBuildersErr).ToNot(HaveOccurred(),
+				fmt.Sprintf("Failed to get Builders for AMD GPU Worker Nodes. Error:\n%v\n", amdNodeBuildersErr))
+
+			Expect(amdNodeBuilders).ToNot(BeEmpty(),
+				"'amdNodeBuilders' can't be empty")
+
+		})
+
+		BeforeEach(func() {})
+
+		AfterEach(func() {})
+
+		AfterAll(func() {})
+
+		It("Check AMD label was added by NFD", func() {
+
+			// The assumption is that all Worker Nodes are equipped with AMD GPUs.
+			amdNFDLabelFound, amdNFDLabelFoundErr := get.LabelPresentOnAllNodes(
+				apiClient, tsp.AMDNFDLabelKey, tsp.AMDNFDLabelValue, inittools.GeneralConfig.WorkerLabelMap)
+
+			Expect(amdNFDLabelFoundErr).ToNot(HaveOccurred(),
```

Comment:
it's ToNot(HaveOccurred() here and later I see you use To(BeNil() for error

I'd suggest using the same method for consistency

Reply:
Done